group/assignment6/move_cars.py

```diff
@@ -0,0 +1,38 @@
+def car_positions(state):
+    """ given state returns array where ith element is the position of ith car """
+    positions = [0] * len(state)
+    for place, car in enumerate(state):
+        positions[car] = place
+    return positions
+
+
+def generate_moves(start, final):
+    """ ith element of start state is the car currently in the ith space
+    ith element of final state is the car that should be in ith space
+    empty space is 0
+    yields optimal sequence of moves in format (car number, old space, new space) """
+
+    pos = car_positions(start)
```

Comment:
not sure why you would need car_positions(start). Isnt it essentially returning the same as start?

Reply:
car_positions returns the inverse of the permutation start

However start in my tests is its own inverse:)